plugins/org.jkiss.dbeaver.ext.postgresql/src/org/jkiss/dbeaver/ext/postgresql/model/PostgreServerExtension.java

```diff
@@ -251,4 +253,16 @@ public interface PostgreServerExtension {
      */
     boolean supportsNativeClient();
 
+
+    /**
+     * Determines if the provided object is a PostgreSQL-specific object (PGObject) like com.amazon.redshift.util.RedshiftObject.
+     */
+    default boolean isPGObject(@Nullable Object object) {
```

Comment:
LEt's move it to PostgreServerExtensionBase

Reply:
We cannot, because `public PostgreServerExtension getServerType()`

Reply:
I mean do not use default implementation in interface but move it in PostgreServerExtensionBase